tests/unit_tests/dft/include/compute_inplace.hpp

```diff
@@ -231,16 +219,10 @@ int DFT_Test<precision, domain>::test_in_place_USM() {
         std::copy(input.begin(), input.end(), inout.begin());
     }
 
-    try {
-        std::vector<sycl::event> dependencies;
-        oneapi::mkl::dft::compute_forward<descriptor_t, FwdInputType>(descriptor, inout.data(),
-                                                                      dependencies)
-            .wait();
-    }
-    catch (oneapi::mkl::unimplemented& e) {
-        std::cout << "Skipping test because: \"" << e.what() << "\"" << std::endl;
-        return test_skipped;
-    }
+    std::vector<sycl::event> no_dependencies;
+    oneapi::mkl::dft::compute_forward<descriptor_t, FwdInputType>(descriptor, inout.data(),
+                                                                  no_dependencies)
+        .wait();
```

Comment:
Now could be a good time to switch to `wait_and_throw()` in all of our tests. This allows us to catch asynchronous errors so this is best practice in SYCL. What do you think?

Reply:
Sounds like a good idea! done in https://github.com/oneapi-src/oneMKL/pull/311/commits/0f0f28745a69f38dfa057fb159a45f98ef2ea6b3